src/shared/containers/GetOrganizations.tsx

```diff
@@ -14,37 +15,78 @@ import {RemoteDataState, AppState} from 'src/types'
 
 // Actions
 import {getOrganizations} from 'src/organizations/actions/thunks'
-import RouteToOrg from './RouteToOrg'
+import {getQuartzMe} from 'src/me/actions/thunks'
+import RouteToOrg from 'src/shared/containers/RouteToOrg'
 
-type ReduxProps = ConnectedProps<typeof connector>
-type Props = ReduxProps
+// Constants
+import {CLOUD} from 'src/shared/constants'
 
-const GetOrganizations: FunctionComponent<Props> = ({status}) => {
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+// Types
+import {Me} from 'src/client/unityRoutes'
+
+const canAccessCheckout = (me: Me): boolean => {
+  if (me?.account?.type === 'pay_as_you_go') {
+    return false
+  }
+  if (me?.account?.type === 'free' && !!me?.isBeta) {
+    return false
+  }
+  return true
+}
```

Comment:
Pretty sure these are the rules, but is there anything else i'm missing in terms of restrictions for allowing access to the checkout page?